packages/devtools_app/test/theme_test.dart

```diff
@@ -12,17 +12,6 @@ import 'package:flutter_test/flutter_test.dart';
 void main() {
   group('Theme', () {
     ThemeData theme;
-    test('can be used without override', () {
-      theme = themeFor(isDarkTheme: true, ideTheme: null);
```

Comment:
instead of removing this test, why don't we just replace `null` with `IdeTheme()`?

Reply:
The goal of the test is to verify everything works with null. We do not need this validation any more.
Other tests here cover non-null cases. Makes sense?

Reply:
In this case an empty IdeTheme would be the equivalent of what used to be null, in terms of testing the "without override" case. The "without override" case is where a user runs DevTools via `dart devtools` instead of using the DevTools embedded in their IDE. So we still want to test the case where IdeTheme does not contain overrides for backgroundColor, foregroundColor, etc.

Reply:
fixed